doc/Reactive.md

```diff
@@ -1,13 +1,67 @@
 # Reactive
+## Concept
+
+When we are asynchronously loading a data, the standard pattern is to use a `Task<T>`. A _task_ represents a data that will be available in the future:
+```csharp
+public async Task<decimal> GetShippingCost(CancellationToken ct)
+{
+	var country = SelectedCountry;
+	var cost = await _shippingService.GetShippingCost(country);
+
+	return cost;
+}
+```
+The issue is that `Task<T>` represents only one value, you must manually re-fetch the data each time one of its dependencies is updated. For instance, here, each time the user updates the selected country, you have to manually re-invoked the `GetShippingCost` and update the UI.
+
+A solution to this would be to use `IObservable<T>` or `IAsyncEnumerable<T>`. Both are representing a stream of value. The example above can be written like this using `IObservable<T>`:
+```csharp
+public IObservable<Country> SelectedCountry { get; }
+
+public IObservable<decimal> ShippingCost => _selectedCountry.SelectAsync(country => _shippingService.GetShippingCost(country));
+```
+Or with `IAsyncEnumerable`:
+```csharp
+public async IAsyncEnumerable<decimal> GetShippingCost([EnumerationCancellation] CancellationToken ct = default)
+{
+	await foreach (var country in SelectedCountry)
+	{
+		yield return await _shippingService.GetShippingCost(country);
+	}
+}
+```
+
+But in both cases, if there is any exception the stream will be broken. This means that for instance in example above, if we cannot compute the shipping cost for a given country for any reason (network issue, invalid country, …) the stream of data will be terminated, and selecting another country won’t have any effect.
+
+Also, when a dependency is being updated and we may need to do some asynchronously work, like update a projection. In our example, we asynchronously get the updated shipping cost when country is changed. From a UI perspective, it would be great to have a visual indication that the shipping cost is being re-computed for the newly chosen country.
+
+Neither `IObservable<T>` nor `IAsyncEnumerable<T>` have such metadata mechanism for produced values, that the purpose if `IFeed<T>`.
+
+With the data, currently `IFeed<T>` does supports 2 mains metadata (named “axis”):
+•	Error: If there is any exception linked to the current data
+•	Progress: We indicates that the current data is transient or final.
+But we do also have a metadata about the data itself:
+•	Some: The _feed_ does have a valid data.
+•	None: There is no data to display. In our example, when you cannot ship to the selected country.
+•	Undefined: There is no info about the data, typically because we are asynchronously loading it.
```

Comment:
The text says there are 2 axis but then lists 5 items in the list. Which are the two axis and how do they correlate to the other items in the list